src/references/ui.md

```diff
@@ -529,6 +529,125 @@ UI.SetCanCursorInteractWithUI(true)
+
+The following table lists the number of images supported per [resolution](../api/cameracapture.md).
+
+| Resolution Size | Maximum Images |
+| --------------- | -------------- |
+| Very Small | 256 |
+| Small | 64 |
+| Medium | 16 |
+| Large | 4 |
+| Very Large | 1 |
+
+### Camera Setup
+
+A [Camera](../api/camera.md) will need to be added to the **Hierarchy** that will be setup just for taking captures. The camera will need certain properties changed so it is setup optimally for creating captures. If any of these properties are not changed, Core will display a warning in the **Event Log** with information on which properties to change.
+
+1. Create a **Client Context** in the **Hierarchy**.
+2. Set the **Position** in the **Properties** window to **X** `-300`, **Z** `350`.
+3. Set the **Field of View** in the **Properties** window to `60`.
+4. Create a **Camera** in the **Client Context** and set the following properties in the **Properties** window:
+
+| Property | Value |
+| -------- | ----- |
+| **Attach To Local Player** | Should be left unchecked. |
+| **Position Offset** | Should be set to Zero. |
+| **Initial Distance** | Should be set to Zero. |
+| **Adjustable Distance** | Should be left unchecked. |
+| **Base Rotation Mode** | Should be set to Camera. |
+| **Rotation Offset** | Should be set to Zero. |
+| **Free Control** | Should be left unchecked. |
+
+### Create UI
+
+To display the camera captures, a **UI Image** will be needed.
+
+1. Create a **UI Container** in the **Client Context**.
+2. Create a **UI Image** in the **UI Container** and name it `Icon`.
+
+### Create Icon Generator Setup
+
+An icon generator will be a small setup made up of a plane, frame, light, dummy object, and a script. It will allow for easy setup of creating icons to use in the UI. This setup would be hidden from players, this is usually placed under the map. The materials, colors, and lighting will be up to the creator to change to get the look they require.
+
+![!Icon Generator](../img/UI/camera_capture_setup.png){: .center loading="lazy" }
+
+#### Create Background
+
+The background will be used behind the object that will be rendered to the UI image. The material and color can be changed to get a different background based on the object being rendered. For example, a different color for epic items, and a different color for legendary items.
+
+1. In **Core Content** search for `Plane 1m - One Sided` and add it into the **Client Context** and rename it to `Background`.
+2. Set the **Position** in the **Properties** window to **X** `601`, **Z** `350`.
+3. Set the **Rotation** in the **Properties** window to **Y** `90`.
+4. Set the **Scale** in the **Properties** window to **X** `6`, **Y** `6`, **Z** `6`.
+
+#### Create Frame
+
+The frame will show in the rendered image. This is optional.
+
+1. In **Core Content** search for `pipe 4 sided thin` and add it into the **Client Context** and rename it to `Frame`.
```

Comment:
Pipe - 4-Sided Thin

not sure if you were lowercasing and removing symbols for easier search

Reply:
Yep, I do a search to see what I can remove from the term that would still just show that item.

Reply:
ok makes sense then. just seemed weird because you wrote "Plane 1m - One Sided" for the previous search

Reply:
Yeah, that was likely a copy and paste. Might change that as well.